x-pack/plugins/apm/public/components/routing/service_detail/index.tsx

```diff
@@ -206,7 +206,7 @@ export const serviceDetail = {
       }),
       children: [
         {
-          path: '/:serviceNodeName/metrics',
+          path: '/services/{serviceName}/nodes/{serviceNodeName}/metrics',
```

Comment:
What happens if a nested route doesn't use the same root as it's parent? Compile time error, runtime error, or nothing?

Reply:
... and what do nested route do for us?

Reply:
Currently, nothing, which is consistent with the React Router API, I think. I'm not opposed to adding a check there, do you think it's worth it?